### DISCUSSION TITLE
希望增加一个跳过片头和忽略片尾的功能

### DISCUSSION THREAD

**Discussion by conglinyizhi** [the1812/Bilibili-Evolved#3008] [功能建议 / Ideas]
是这样的，大多数 UP 主为了自己的品牌吧，做了自己的片头和片尾，但是这些东西对于我来说不怎么喜欢，所以我希望可以跳过去，播放到片尾的时候直接视频结束，有的片头和片尾还挺……社死的

**Comment by Geeyun-JY3** [comment]
这怎么跳……各 UP 片头片尾的时长又不一样，你不如在快捷键扩展设置个大概的长跳跃秒数

**Reply by Geeyun-JY3** [reply]
做 b 站的 SponsorBlock？

**Reply by conglinyizhi** [reply]
> 做 b 站的 SponsorBlock？

看了一下介绍，好像差不多是那个意思……

**Reply by conglinyizhi** [reply]
就是通过 UP 主判断的，固定的片头和片尾跳过，还有必剪的片尾……

**Reply by Geeyun-JY3** [reply]
我觉得难度、工程量都不小，更何况 SponsorBlock 的标记可是靠众包的……这又有额外的成本

**Reply by timongh** [reply]
倒是可以是自己手动标记时间，这样就没有额外成本了

**Comment by RingingVolcano** [comment]
感觉这个工程量挺大的。。。

**Comment by Geeyun-JY3** [comment]
关联：#1629

**Comment by the1812** [comment]
我要不加个长倒退得了, 根据 UP 主判断不太现实

**Reply by Geeyun-JY3** [reply]
不是有吗（直球

不如再加个跳转至视频结尾？

**Reply by the1812** [reply]
噢 太久远了我都忘了

**Comment by timongh** [comment]
我的意思是，由用户自己标注自己关注的 up。插件就本地存储一下标注信息，有标注的就跳转，没标注就不跳。
